1001-alerts-design-guidance.md

```diff
@@ -0,0 +1,56 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-21
+updated: 2024-10-21
+---
+
+# Alert Design Guidance
+
+The [alert](https://clarity.design/documentation/alert) component displays banners that demand the user's attention for important messages.
+
+## Guidance
+
+### App-Level Alerts
+
+App-level alerts support only error, warning, and info statuses.
+
+- **Must** be used exclusively for the global context of an application.
+- **Must** be placed at the top of *all* content. 
+- **Must not** be used to convey the success of an operation.
```

Comment:
```suggestion
- Design teams **must not** use alerts to convey the success of an operation, as this can dilute their effectiveness in communicating important information.
```